cli/tests/pcluster/cli_commands/configure/test_pcluster_configure.py

```diff
@@ -362,15 +362,18 @@ def __init__(self, aws_region_name, key, scheduler):
         self.input_list = [] if aws_region_name is None else [aws_region_name]
         self.input_list.extend([key, scheduler])
 
-    def add_first_flow(self, op_sys, min_size, max_size, head_node_instance, compute_instance):
+    def add_first_flow(self, op_sys, max_size, head_node_instance, compute_instance):
         if self.is_not_aws_batch:
             self.input_list.append(op_sys)
         self.input_list.append(head_node_instance)
         if self.is_not_aws_batch:
-            self.input_list.extend(["1", "1"])
+            self.input_list.append("1")
+        self.input_list.append("myqueue")
+        if self.is_not_aws_batch:
+            self.input_list.append("1")
         if self.is_not_aws_batch:
```

Comment:
minor: you can unify two consecutive blocks
```
if self.is_not_aws_batch:
    self.input_list.append("1")
    self.input_list.append(compute_instance)
```

Reply:
Done